packages/documentation/package.json

```diff
@@ -6,18 +6,16 @@
     "start": "astro dev",
     "build": "astro build",
     "preview": "astro preview",
-    "astro": "astro",
-    "docs:graphql:generate:backend": "./node_modules/.bin/graphql-markdown ../backend/src/graphql/schema.graphql > src/schema-dumps/backend_raw_schema.md",
-    "docs:graphql:generate:auth": "./node_modules/.bin/graphql-markdown ../auth/src/graphql/schema.graphql > src/schema-dumps/auth_raw_schema.md"
```

Comment:
@huijing

> Also, how does this work, do the markdown files get generated when the build runs or? How do people update the docs when the schema changes?

It looks like they are generated on the build. In that case, I propose 
- we don't commit these generated files into the repo and
- we remove the CI step `pnpm --filter documentation docs:graphql:generate` 

Otherwise, we should add a command `docs:graphql:generate`, which will fail a build if the generated files are outdated compared to the graphql schema

Reply:
If we don't exclude them, we _could_ get weird diffs in PRs as a result of people building the docs locally then pushing up their changes. I was just thinking of treating them as output to a build step, similar to how we ignore `build` folders

Reply:
They need to be put into the "content" folder before the build step, let me see if there's a hook for that in Astro.

Reply:
Removed the files from content folder, added them to gitignore, and now they get generated every time the "build" command is issued.